Packs/FeedAutofocus/Integrations/FeedAutofocus/FeedAutofocus.py

```diff
@@ -76,48 +121,82 @@ def find_indicator_type(self, indicator):
         Returns:
             str. The type of the indicator.
         """
-        if re.match(ipv4cidrRegex, indicator):
-            return FeedIndicatorType.CIDR
 
-        if re.match(ipv6cidrRegex, indicator):
-            return FeedIndicatorType.IPv6CIDR
+        # trying to catch X.X.X.X:portNum
+        if ':' in indicator and '/' not in indicator:
+            sub_indicator = indicator.split(':', 1)[0]
+            ip_type = self.is_ip_type(sub_indicator)
+            if ip_type:
+                return ip_type
 
-        if re.match(ipv4Regex, indicator):
-            return FeedIndicatorType.IP
+        ip_type = self.is_ip_type(indicator)
+        if ip_type:
+            # catch URLs of type X.X.X.X/path/url or X.X.X.X:portNum/path/url
+            if '/' in indicator and (ip_type not in [FeedIndicatorType.IPv6CIDR, FeedIndicatorType.CIDR]):
+                return FeedIndicatorType.URL
 
-        if re.match(ipv6Regex, indicator):
-            return FeedIndicatorType.IPv6
+            else:
+                return ip_type
 
         elif re.match(sha256Regex, indicator):
             return FeedIndicatorType.File
 
-        # in AutoFocus, URLs include '/' character while domains do not.
+        # in AutoFocus, URLs include a path while domains do not - so '/' is a good sign for us to catch URLs.
         elif '/' in indicator:
             return FeedIndicatorType.URL
 
         else:
             return FeedIndicatorType.Domain
 
+    def resolve_ip_address(self, ip):
```

Comment:
why do we need this exactly?

Reply:
@avidan-H and I talked about this - things like this:
X.X.X.X/rest/of/path
are recognized as URLs in AutoFocus - we thought we could resolve the IP address to show a more standard URL